### ISSUE TITLE
Greenhouse doesn't work in the Nether

### ISSUE THREAD

**Issue by Karpatz** [tastybento/greenhouses#2]
Greenhouse doesn't work in the Nether, bone meal is consumed in the hooper but they are no loot in the greenhouse.

**Comment by tastybento** [comment]
Ok, I'll look into this. Can you also check a few other nether items? Water or ice should disappear if a gh is broken. Also, water should not flow out of a gh.

**Comment by Karpatz** [comment]
Ok i try this .

**Comment by Karpatz** [comment]
So i try , when i brake the gh , water flow out and snow stay , i check my config gh :  

### Default settings for greenhouse actions

   # Allow lava or water to flow out of a greenhouse, e.g. through the door, floor
   allowflowout: false
   # Allow lava or water to flow into a greenhouse, e.g., through the door
   allowflowin: false

**Comment by tastybento** [comment]
Try this build and see if it fixes all your issues:
https://github.com/tastybento/greenhouses/releases/tag/3.0

**Comment by Karpatz** [comment]
Work great !  The water not flow , they are loot and mob, and when we brake the gc the snow ice and water disappear thx a lot !

**Comment by Karpatz** [comment]
I found a bug when you change the biome ,when i put water i think it's reset and  the water disappear.
exemple: i have made a gh plain, i put water for another biome, i want ocean and the water disappear , and broke the gh

**Comment by tastybento** [comment]
Ahhh. Hmm.  Here's what I think is happening.  The rules for the greenhouse are being broken so it is being reverted to nether.  When you tried the new version did you delete biomes.yml? You want to make sure there is no water coverage : 0 in there. When you use /g make sure the biome doesn't say water not allowed.

**Comment by Karpatz** [comment]
Ok i see, but for the challenges, if a player start with the nether island, and want to do for example the slimeballfarmer, he need the swamp biome in the nether, so he c'ant do it in the nether. But anyway , i think i must made new challenge , and new gh biomes for this feature but have no idea a this time ^^ . But i have question there is a challenge permission ?

**Comment by tastybento** [comment]
> But i have question there is a challenge permission ?

I don't understand the question. Can you rephrase?

**Comment by tastybento** [comment]
> Ok i see, but for the challenges, if a player start with the nether island, and want to do for example the slimeballfarmer, he need the swamp biome in the nether, so he c'ant do it in the nether.

There's something wrong with the water detection, and it's too difficult to make this biome in the Nether. Let me look into a way to make this possible.

**Comment by Karpatz** [comment]
Excuse me for my bad english ^^  I mean , If his possible to make challenges with a permission with acidisland plugin ? (I want to create a degree (NetherVIp) with custom challenges with acidisland. but not on another server) but if you find a solution to gh to make biome with watter in the nether, it's good for me ^^
